
PMID- 23894639
OWN - NLM
STAT- MEDLINE
DA  - 20130729
DCOM- 20140220
LR  - 20141113
IS  - 1932-6203 (Electronic)
IS  - 1932-6203 (Linking)
VI  - 8
IP  - 7
DP  - 2013
TI  - Has large-scale named-entity network analysis been resting on a flawed
      assumption?
PG  - e70299
LID - 10.1371/journal.pone.0070299 [doi]
AB  - The assumption that a name uniquely identifies an entity introduces two types of 
      errors: splitting treats one entity as two or more (because of name variants);
      lumping treats multiple entities as if they were one (because of shared names).
      Here we investigate the extent to which splitting and lumping affect
      commonly-used measures of large-scale named-entity networks within two
      disambiguated bibliographic datasets: one for co-author names in biomedicine
      (PubMed, 2003-2007); the other for co-inventor names in U.S. patents (USPTO,
      2003-2007). In both cases, we find that splitting has relatively little effect,
      whereas lumping has a dramatic effect on network measures. For example, in the
      biomedical co-authorship network, lumping (based on last name and both initials) 
      drives several measures down: the global clustering coefficient by a factor of 4 
      (from 0.265 to 0.066); degree assortativity by a factor of approximately 13 (from
      0.763 to 0.06); and average shortest path by a factor of 1.3 (from 5.9 to 4.5).
      These results can be explained in part by the fact that lumping artificially
      creates many intransitive relationships and high-degree vertices. This effect of 
      lumping is much less dramatic but persists with measures that give less weight to
      high-degree vertices, such as the mean local clustering coefficient and log-based
      degree assortativity. Furthermore, the log-log distribution of collaborator
      counts follows a much straighter line (power law) with splitting and lumping
      errors than without, particularly at the low and the high counts. This suggests
      that part of the power law often observed for collaborator counts in science and 
      technology reflects an artifact: name ambiguity.
FAU - Fegley, Brent D
AU  - Fegley BD
AD  - Graduate School of Library and Information Science, University of Illinois at
      Urbana-Champaign, Champaign, Illinois, United States of America.
FAU - Torvik, Vetle I
AU  - Torvik VI
LA  - eng
PT  - Journal Article
PT  - Research Support, Non-U.S. Gov't
DEP - 20130724
PL  - United States
TA  - PLoS One
JT  - PloS one
JID - 101285081
SB  - IM
MH  - Algorithms
MH  - *Names
MH  - Publications
PMC - PMC3722140
OID - NLM: PMC3722140
EDAT- 2013/07/31 06:00
MHDA- 2014/02/22 06:00
CRDT- 2013/07/30 06:00
PHST- 2013 [ppublish]
PHST- 2013/02/09 [received]
PHST- 2013/06/18 [accepted]
PHST- 2013/07/24 [epublish]
AID - 10.1371/journal.pone.0070299 [doi]
AID - PONE-D-13-06447 [pii]
PST - epublish
SO  - PLoS One. 2013 Jul 24;8(7):e70299. doi: 10.1371/journal.pone.0070299. Print 2013.

PMID- 22427989
OWN - NLM
STAT- MEDLINE
DA  - 20120319
DCOM- 20120820
LR  - 20141019
IS  - 1932-6203 (Electronic)
IS  - 1932-6203 (Linking)
VI  - 7
IP  - 3
DP  - 2012
TI  - MicroRNA expression is down-regulated and reorganized in prefrontal cortex of
      depressed suicide subjects.
PG  - e33201
LID - 10.1371/journal.pone.0033201 [doi]
AB  - BACKGROUND: Recent studies suggest that alterations in expression of genes,
      including those which regulate neural and structural plasticity, may be crucial
      in the pathogenesis of depression. MicroRNAs (miRNAs) are newly discovered
      regulators of gene expression that have recently been implicated in a variety of 
      human diseases, including neuropsychiatric diseases. METHODOLOGY/PRINCIPAL
      FINDINGS: The present study was undertaken to examine whether the miRNA network
      is altered in the brain of depressed suicide subjects. Expression of miRNAs was
      measured in prefrontal cortex (Brodmann Area 9) of antidepressant-free depressed 
      suicide (n = 18) and well-matched non-psychiatric control subjects (n = 17) using
      multiplex RT-PCR plates. We found that overall miRNA expression was significantly
      and globally down-regulated in prefrontal cortex of depressed suicide subjects.
      Using individual tests of statistical significance, 21 miRNAs were significantly 
      decreased at p = 0.05 or better. Many of the down-regulated miRNAs were encoded
      at nearby chromosomal loci, shared motifs within the 5'-seeds, and shared
      putative mRNA targets, several of which have been implicated in depression. In
      addition, a set of 29 miRNAs, whose expression was not pairwise correlated in the
      normal controls, showed a high degree of co-regulation across individuals in the 
      depressed suicide group. CONCLUSIONS/SIGNIFICANCE: The findings show widespread
      changes in miRNA expression that are likely to participate in pathogenesis of
      major depression and/or suicide. Further studies are needed to identify whether
      the miRNA changes lead to altered expression of prefrontal cortex mRNAs, either
      directly (by acting as miRNA targets) or indirectly (e.g., by affecting
      transcription factors).
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry, Psychiatric Institute, University of Illinois at
      Chicago, Chicago, Illinois, United States of America.
FAU - Lugli, Giovanni
AU  - Lugli G
FAU - Rizavi, Hooriyah S
AU  - Rizavi HS
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Turecki, Gustavo
AU  - Turecki G
FAU - Dwivedi, Yogesh
AU  - Dwivedi Y
LA  - eng
GR  - MOP-171859/Canadian Institutes of Health Research/Canada
GR  - MOP-190734/Canadian Institutes of Health Research/Canada
GR  - R01 MH082802/MH/NIMH NIH HHS/United States
GR  - R21MH081099/MH/NIMH NIH HHS/United States
GR  - R21MH091509/MH/NIMH NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20120309
PL  - United States
TA  - PLoS One
JT  - PloS one
JID - 101285081
RN  - 0 (DGCR8 protein, human)
RN  - 0 (DNA Primers)
RN  - 0 (MicroRNAs)
RN  - 0 (Proteins)
RN  - EC 3.1.26.3 (DICER1 protein, human)
RN  - EC 3.1.26.3 (DROSHA protein, human)
RN  - EC 3.1.26.3 (Ribonuclease III)
RN  - EC 3.6.4.13 (DEAD-box RNA Helicases)
SB  - IM
MH  - Adult
MH  - Blotting, Western
MH  - Case-Control Studies
MH  - DEAD-box RNA Helicases/metabolism
MH  - DNA Primers/genetics
MH  - Depression/genetics/*metabolism
MH  - Female
MH  - Gene Expression Regulation/genetics/*physiology
MH  - Humans
MH  - Male
MH  - MicroRNAs/*metabolism
MH  - Middle Aged
MH  - Multiplex Polymerase Chain Reaction
MH  - Prefrontal Cortex/*metabolism
MH  - Proteins/metabolism
MH  - Quebec
MH  - Reverse Transcriptase Polymerase Chain Reaction
MH  - Ribonuclease III/metabolism
MH  - Statistics, Nonparametric
MH  - *Suicide
PMC - PMC3302855
OID - NLM: PMC3302855
EDAT- 2012/03/20 06:00
MHDA- 2012/08/21 06:00
CRDT- 2012/03/20 06:00
PHST- 2011/04/04 [received]
PHST- 2012/02/13 [accepted]
PHST- 2012/03/09 [epublish]
AID - 10.1371/journal.pone.0033201 [doi]
AID - PONE-D-11-06021 [pii]
PST - ppublish
SO  - PLoS One. 2012;7(3):e33201. doi: 10.1371/journal.pone.0033201. Epub 2012 Mar 9.

PMID- 14728536
OWN - NLM
STAT- MEDLINE
DA  - 20040119
DCOM- 20041214
LR  - 20090309
IS  - 1942-597X (Electronic)
IS  - 1559-4076 (Linking)
DP  - 2003
TI  - A probabilistic similarity metric for Medline records: a model for author name
      disambiguation.
PG  - 1033
AB  - We present a model for automatically generating training sets and estimating the 
      probability that a pair of Medline records sharing a last and first name initial 
      are authored by the same individual, based on shared title words, journal name,
      co-authors, medical subject headings, language, and affiliation, as well as
      distinctive features of the name itself (i.e., presence of middle initial,
      suffix, and prevalence in Medline).
FAU - Torvik, Vetle I
AU  - Torvik VI
AD  - Department of Psychiatry, University of Illinois at Chicago, IL, USA.
FAU - Weeber, Marc
AU  - Weeber M
FAU - Swanson, Don R
AU  - Swanson DR
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
LA  - eng
PT  - Journal Article
PL  - United States
TA  - AMIA Annu Symp Proc
JT  - AMIA ... Annual Symposium proceedings / AMIA Symposium. AMIA Symposium
JID - 101209213
SB  - IM
MH  - Authorship
MH  - *Bibliometrics
MH  - *MEDLINE
MH  - Medical Subject Headings
MH  - *Names
MH  - Probability
PMC - PMC1480109
OID - NLM: PMC1480109
EDAT- 2004/01/20 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/01/20 05:00
AID - D030002922 [pii]
PST - ppublish
SO  - AMIA Annu Symp Proc. 2003:1033.

PMID- 15453917
OWN - NLM
STAT- MEDLINE
DA  - 20041021
DCOM- 20041202
LR  - 20140608
IS  - 1471-2105 (Electronic)
IS  - 1471-2105 (Linking)
VI  - 5
DP  - 2004 Sep 28
TI  - A population-based statistical approach identifies parameters characteristic of
      human microRNA-mRNA interactions.
PG  - 139
AB  - BACKGROUND: MicroRNAs are approximately 17-24 nt. noncoding RNAs found in all
      eukaryotes that degrade messenger RNAs via RNA interference (if they bind in a
      perfect or near-perfect complementarity to the target mRNA), or arrest
      translation (if the binding is imperfect). Several microRNA targets have been
      identified in lower organisms, but only one mammalian microRNA target has yet
      been validated experimentally. RESULTS: We carried out a population-wide
      statistical analysis of how human microRNAs interact complementarily with human
      mRNAs, looking for characteristics that differ significantly as compared with
      scrambled control sequences. These characteristics were used to identify a set of
      71 outlier mRNAs unlikely to have been hit by chance. Unlike the case in C.
      elegans and Drosophila, many human microRNAs exhibited long exact matches (10 or 
      more bases in a row), up to and including perfect target complementarity. Human
      microRNAs hit outlier mRNAs within the protein coding region about 2/3 of the
      time. And, the stretches of perfect complementarity within microRNA hits onto
      outlier mRNAs were not biased near the 5'-end of the microRNA. In several cases, 
      an individual microRNA hit multiple mRNAs that belonged to the same functional
      class. CONCLUSIONS: The analysis supports the notion that sequence
      complementarity is the basis by which microRNAs recognize their biological
      targets, but raises the possibility that human microRNA-mRNA target interactions 
      follow different rules than have been previously characterized in Drosophila and 
      C. elegans.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - University of Illinois at Chicago, UIC Psychiatric Institute, MC 912, 1601 W,
      Taylor Street, room 285 Chicago, IL 60612, USA. smalheiser@psych.uic.edu
FAU - Torvik, Vetle I
AU  - Torvik VI
LA  - eng
GR  - DA15450/DA/NIDA NIH HHS/United States
GR  - LM07292/LM/NLM NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, U.S. Gov't, P.H.S.
DEP - 20040928
PL  - England
TA  - BMC Bioinformatics
JT  - BMC bioinformatics
JID - 100965194
RN  - 0 (MicroRNAs)
RN  - 0 (RNA, Messenger)
SB  - IM
MH  - Computational Biology/methods/statistics & numerical data
MH  - Genetics, Population/*statistics & numerical data
MH  - Humans
MH  - MicroRNAs/genetics/*metabolism
MH  - RNA, Messenger/genetics/*metabolism
PMC - PMC523849
OID - NLM: PMC523849
EDAT- 2004/09/30 05:00
MHDA- 2004/12/16 09:00
CRDT- 2004/09/30 05:00
PHST- 2004/04/12 [received]
PHST- 2004/09/28 [accepted]
PHST- 2004/09/28 [aheadofprint]
AID - 10.1186/1471-2105-5-139 [doi]
AID - 1471-2105-5-139 [pii]
PST - epublish
SO  - BMC Bioinformatics. 2004 Sep 28;5:139.

PMID- 15922829
OWN - NLM
STAT- MEDLINE
DA  - 20050530
DCOM- 20050908
LR  - 20071114
IS  - 0168-9525 (Print)
IS  - 0168-9525 (Linking)
VI  - 21
IP  - 6
DP  - 2005 Jun
TI  - Mammalian microRNAs derived from genomic repeats.
PG  - 322-6
AB  - In this article, we show that a subset of conventional mammalian microRNAs is
      derived from LINE-2 transposable elements and other genome repeats. These
      repeat-derived microRNAs arise from conventional precursor hairpins and are
      distinct from the rasiRNAs, which appear to be processed from long
      double-stranded RNA precursors. The insertion of transposable elements into new
      genomic sites appears to be one of the driving-forces that create new microRNAs
      during mammalian evolution. Two of the LINE-2-derived microRNAs exhibit perfect
      complementarity to a large family of mRNA and EST transcripts that contain
      portions of MIR and other LINE-2 elements in their 3'-untranslated regions.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - University of Illinois at Chicago, UIC Psychiatric Institute, MC 912, 1601 W.
      Taylor Street, Chicago, IL 60612, USA. smalheiser@psych.uic.edu
FAU - Torvik, Vetle I
AU  - Torvik VI
LA  - eng
GR  - DA 15450/DA/NIDA NIH HHS/United States
GR  - LM 07292/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, U.S. Gov't, P.H.S.
PL  - England
TA  - Trends Genet
JT  - Trends in genetics : TIG
JID - 8507085
RN  - 0 (DNA Transposable Elements)
RN  - 0 (MicroRNAs)
RN  - 0 (RNA, Messenger)
SB  - IM
MH  - Animals
MH  - Base Sequence
MH  - Computational Biology
MH  - DNA Transposable Elements
MH  - Expressed Sequence Tags
MH  - Humans
MH  - Long Interspersed Nucleotide Elements
MH  - Mice
MH  - MicroRNAs/*genetics
MH  - Molecular Sequence Data
MH  - RNA, Messenger/genetics
MH  - Rats
MH  - *Repetitive Sequences, Nucleic Acid
EDAT- 2005/06/01 09:00
MHDA- 2005/09/09 09:00
CRDT- 2005/06/01 09:00
PHST- 2004/03/04 [received]
PHST- 2005/01/17 [revised]
PHST- 2005/04/08 [accepted]
AID - S0168-9525(05)00111-3 [pii]
AID - 10.1016/j.tig.2005.04.008 [doi]
PST - ppublish
SO  - Trends Genet. 2005 Jun;21(6):322-6.

PMID- 16817965
OWN - NLM
STAT- PubMed-not-MEDLINE
DA  - 20060904
DCOM- 20070724
LR  - 20091118
IS  - 1747-5333 (Electronic)
IS  - 1747-5333 (Linking)
VI  - 1
DP  - 2006
TI  - Collaborative development of the Arrowsmith two node search interface designed
      for laboratory investigators.
PG  - 8
AB  - Arrowsmith is a unique computer-assisted strategy designed to assist
      investigators in detecting biologically-relevant connections between two
      disparate sets of articles in Medline. This paper describes how an
      inter-institutional consortium of neuroscientists used the UIC Arrowsmith web
      interface http://arrowsmith.psych.uic.edu in their daily work and guided the
      development, refinement and expansion of the system into a suite of tools
      intended for use by the wider scientific community.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry and Psychiatric Institute, University of Illinois,
      MC912, 1601 W, Taylor Street, Chicago, IL 60612, USA. neils@uic.edu
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Bischoff-Grethe, Amanda
AU  - Bischoff-Grethe A
FAU - Burhans, Lauren B
AU  - Burhans LB
FAU - Gabriel, Michael
AU  - Gabriel M
FAU - Homayouni, Ramin
AU  - Homayouni R
FAU - Kashef, Alireza
AU  - Kashef A
FAU - Martone, Maryann E
AU  - Martone ME
FAU - Perkins, Guy A
AU  - Perkins GA
FAU - Price, Diana L
AU  - Price DL
FAU - Talk, Andrew C
AU  - Talk AC
FAU - West, Ruth
AU  - West R
LA  - eng
PT  - Journal Article
DEP - 20060703
PL  - England
TA  - J Biomed Discov Collab
JT  - Journal of biomedical discovery and collaboration
JID - 101258054
PMC - PMC1559644
OID - NLM: PMC1559644
EDAT- 2006/07/05 09:00
MHDA- 2006/07/05 09:01
CRDT- 2006/07/05 09:00
PHST- 2006/02/07 [received]
PHST- 2006/07/03 [accepted]
PHST- 2006/07/03 [aheadofprint]
AID - 1747-5333-1-8 [pii]
AID - 10.1186/1747-5333-1-8 [doi]
PST - epublish
SO  - J Biomed Discov Collab. 2006 Jul 3;1:8.

PMID- 16914224
OWN - NLM
STAT- MEDLINE
DA  - 20060918
DCOM- 20061211
LR  - 20071203
IS  - 0168-9525 (Print)
IS  - 0168-9525 (Linking)
VI  - 22
IP  - 10
DP  - 2006 Oct
TI  - Alu elements within human mRNAs are probable microRNA targets.
PG  - 532-6
AB  - Recently, we reported that four microRNAs show perfect complementarity with
      MIR/LINE-2 elements within human mRNAs. This finding raises the question of
      whether microRNAs might also target other genomic repeats and transposable
      elements. Here, we demonstrate that almost 30 human microRNAs exhibit typical
      short-seed complementarity with a specific site within Alu elements that is
      highly conserved within 3' untranslated regions of human mRNAs. The results
      suggest that at least some Alu elements within human mRNAs serve as microRNA
      targets.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - University of Illinois-Chicago, UIC Psychiatric Institute MC912, 1601 W. Taylor
      Street, Chicago, IL 60612, USA. neils@uic.edu
FAU - Torvik, Vetle I
AU  - Torvik VI
LA  - eng
GR  - DA 15450/DA/NIDA NIH HHS/United States
GR  - LM 07292/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20060817
PL  - England
TA  - Trends Genet
JT  - Trends in genetics : TIG
JID - 8507085
RN  - 0 (3' Untranslated Regions)
RN  - 0 (MicroRNAs)
RN  - 0 (RNA, Messenger)
SB  - IM
MH  - 3' Untranslated Regions
MH  - *Alu Elements
MH  - Base Sequence
MH  - Humans
MH  - MicroRNAs/genetics/*metabolism
MH  - Models, Genetic
MH  - Molecular Sequence Data
MH  - RNA, Messenger/*chemistry
MH  - Sequence Alignment
EDAT- 2006/08/18 09:00
MHDA- 2006/12/12 09:00
CRDT- 2006/08/18 09:00
PHST- 2005/12/01 [received]
PHST- 2006/05/24 [revised]
PHST- 2006/08/02 [accepted]
PHST- 2006/08/17 [aheadofprint]
AID - S0168-9525(06)00260-5 [pii]
AID - 10.1016/j.tig.2006.08.007 [doi]
PST - ppublish
SO  - Trends Genet. 2006 Oct;22(10):532-6. Epub 2006 Aug 17.

PMID- 16982707
OWN - NLM
STAT- MEDLINE
DA  - 20061106
DCOM- 20070105
LR  - 20091104
IS  - 1367-4811 (Electronic)
IS  - 1367-4803 (Linking)
VI  - 22
IP  - 22
DP  - 2006 Nov 15
TI  - ADAM: another database of abbreviations in MEDLINE.
PG  - 2813-8
AB  - MOTIVATION: Abbreviations are an important type of terminology in the biomedical 
      domain. Although several groups have already created databases of biomedical
      abbreviations, these are either not public, or are not comprehensive, or focus
      exclusively on acronym-type abbreviations. We have created another abbreviation
      database, ADAM, which covers commonly used abbreviations and their definitions
      (or long-forms) within MEDLINE titles and abstracts, including both acronym and
      non-acronym abbreviations. RESULTS: A model of recognizing abbreviations and
      their long-forms from titles and abstracts of MEDLINE (2006 baseline) was
      employed. After grouping morphological variants, 59 405 abbreviation/long-form
      pairs were identified. ADAM shows high precision (97.4%) and includes most of the
      frequently used abbreviations contained in the Unified Medical Language System
      (UMLS) Lexicon and the Stanford Abbreviation Database. Conversely, one-third of
      abbreviations in ADAM are novel insofar as they are not included in either
      database. About 19% of the novel abbreviations are non-acronym-type and these
      cover at least seven different types of short-form/long-form pairs. AVAILABILITY:
      A free, public query interface to ADAM is available at
      http://arrowsmith.psych.uic.edu, and the entire database can be downloaded as a
      text file.
FAU - Zhou, Wei
AU  - Zhou W
AD  - Department of Psychiatry and Psychiatric Institute, MC912, University of Illinois
      at Chicago Chicago, IL 60612, USA.
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
LA  - eng
GR  - LM007292/LM/NLM NIH HHS/United States
GR  - LM08364/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20060918
PL  - England
TA  - Bioinformatics
JT  - Bioinformatics (Oxford, England)
JID - 9808944
SB  - IM
MH  - Animals
MH  - Databases as Topic
MH  - *Databases, Bibliographic
MH  - *Databases, Factual
MH  - Dictionaries as Topic
MH  - Humans
MH  - Information Storage and Retrieval
MH  - Internet
MH  - Natural Language Processing
MH  - Pattern Recognition, Automated
MH  - Software
MH  - Subject Headings
MH  - *Terminology as Topic
MH  - Unified Medical Language System
MH  - User-Computer Interface
EDAT- 2006/09/20 09:00
MHDA- 2007/01/06 09:00
CRDT- 2006/09/20 09:00
PHST- 2006/09/18 [aheadofprint]
AID - btl480 [pii]
AID - 10.1093/bioinformatics/btl480 [doi]
PST - ppublish
SO  - Bioinformatics. 2006 Nov 15;22(22):2813-8. Epub 2006 Sep 18.

PMID- 16957371
OWN - NLM
STAT- MEDLINE
DA  - 20060907
DCOM- 20060928
LR  - 20071114
IS  - 1064-3745 (Print)
IS  - 1064-3745 (Linking)
VI  - 342
DP  - 2006
TI  - Complications in mammalian microRNA target prediction.
PG  - 115-27
AB  - In this chapter, we review evidence that at least three different types of
      microRNA (miRNA)-messenger RNA (mRNA) target interactions exist in mammals: short
      seeds, long seeds, and "perfect" hits (allowing G:U matches). Because new types
      of miRNAs are still being discovered, this list may not yet be complete.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry and UIC Psychiatric Institute, University of
      Illinois-Chicago, IL, USA.
FAU - Torvik, Vetle I
AU  - Torvik VI
LA  - eng
GR  - DA15450/DA/NIDA NIH HHS/United States
GR  - LM07292/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Review
PL  - United States
TA  - Methods Mol Biol
JT  - Methods in molecular biology (Clifton, N.J.)
JID - 9214969
RN  - 0 (3' Untranslated Regions)
RN  - 0 (MicroRNAs)
RN  - 0 (RNA, Messenger)
SB  - IM
MH  - 3' Untranslated Regions
MH  - Animals
MH  - Databases, Genetic
MH  - Expressed Sequence Tags
MH  - Long Interspersed Nucleotide Elements
MH  - Mammals/classification/*genetics/*metabolism
MH  - MicroRNAs/chemistry/*genetics/*metabolism
MH  - RNA Interference
MH  - RNA, Messenger/chemistry/genetics/metabolism
MH  - Software
RF  - 32
EDAT- 2006/09/08 09:00
MHDA- 2006/09/29 09:00
CRDT- 2006/09/08 09:00
AID - 1-59745-123-1:115 [pii]
AID - 10.1385/1-59745-123-1:115 [doi]
PST - ppublish
SO  - Methods Mol Biol. 2006;342:115-27.

PMID- 17463015
OWN - NLM
STAT- MEDLINE
DA  - 20070724
DCOM- 20070827
LR  - 20091104
IS  - 1367-4811 (Electronic)
IS  - 1367-4803 (Linking)
VI  - 23
IP  - 13
DP  - 2007 Jul 1
TI  - A quantitative model for linking two disparate sets of articles in MEDLINE.
PG  - 1658-65
AB  - BACKGROUND: Identifying information that implicitly links two disparate sets of
      articles is a fundamental and intuitive data mining strategy that can help
      investigators address real scientific questions. The Arrowsmith two-node search
      finds title words and phrases (so-called B-terms) that are shared across two sets
      of articles within MEDLINE and displays them in a manner that facilitates human
      assessment. A serious stumbling-block has been the lack of a quantitative model
      for predicting which of the hundreds if not thousands of B-terms computed for a
      given search are most likely to be relevant to the investigator.
      METHODOLOGY/PRINCIPAL FINDINGS: Using a public two-node search interface, field
      testers devised a set of two-node searches under real life conditions and a
      certain number of B-terms were marked relevant. These were employed as 'gold
      standards;' each B-term was characterized according to eight complementary
      features that were strongly correlated with relevance. A logistic regression
      model was developed that permits one to estimate the probability of relevance for
      each B-term, to rank B-terms according to their likely relevance, and to estimate
      the overall number of relevant B-terms inherent in a given two-node search.
      CONCLUSIONS/SIGNIFICANCE: The model greatly simplifies and streamlines the
      process of carrying out a two-node search, and may be applicable to a number of
      other literature-based discovery applications, including the so-called one-node
      search and related gene-centric strategies that incorporate implicit links to
      predict how genes may be related to each other and to human diseases. This should
      encourage much wider exploration of text mining for implicit information among
      the general scientific community. AVAILABILITY: Two-node searches can be carried 
      out freely at http://arrowsmith.psych.uic.edu. SUPPLEMENTARY INFORMATION:
      Supplementary data are available at Bioinformatics online.
FAU - Torvik, Vetle I
AU  - Torvik VI
AD  - Department of Psychiatry and Psychiatric Institute (MC912), University of
      Illinois-Chicago, Chicago, IL 60612, USA.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
LA  - eng
GR  - LM007292/LM/NLM NIH HHS/United States
GR  - LM08364/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20070426
PL  - England
TA  - Bioinformatics
JT  - Bioinformatics (Oxford, England)
JID - 9808944
SB  - IM
MH  - *Algorithms
MH  - *Artificial Intelligence
MH  - Information Storage and Retrieval/*methods
MH  - *MEDLINE
MH  - *Natural Language Processing
MH  - Pattern Recognition, Automated/*methods
MH  - *Periodicals as Topic
MH  - *Terminology as Topic
MH  - Vocabulary, Controlled
EDAT- 2007/04/28 09:00
MHDA- 2007/08/28 09:00
CRDT- 2007/04/28 09:00
PHST- 2007/04/26 [aheadofprint]
AID - btm161 [pii]
AID - 10.1093/bioinformatics/btm161 [doi]
PST - ppublish
SO  - Bioinformatics. 2007 Jul 1;23(13):1658-65. Epub 2007 Apr 26.

PMID- 18279519
OWN - NLM
STAT- PubMed-not-MEDLINE
DA  - 20080328
DCOM- 20110714
LR  - 20111121
IS  - 1747-5333 (Electronic)
IS  - 1747-5333 (Linking)
VI  - 3
DP  - 2008
TI  - Anne O'Tate: A tool to support user-driven summarization, drill-down and browsing
      of PubMed search results.
PG  - 2
LID - 10.1186/1747-5333-3-2 [doi]
AB  - BACKGROUND: PubMed is designed to provide rapid, comprehensive retrieval of
      papers that discuss a given topic. However, because PubMed does not organize the 
      search output further, it is difficult for users to grasp an overview of the
      retrieved literature according to non-topical dimensions, to drill-down to find
      individual articles relevant to a particular individual's need, or to browse the 
      collection. RESULTS: In this paper, we present Anne O'Tate, a web-based tool that
      processes articles retrieved from PubMed and displays multiple aspects of the
      articles to the user, according to pre-defined categories such as the "most
      important" words found in titles or abstracts; topics; journals; authors;
      publication years; and affiliations. Clicking on a given item opens a new window 
      that displays all papers that contain that item. One can navigate by drilling
      down through the categories progressively, e.g., one can first restrict the
      articles according to author name and then restrict that subset by affiliation.
      Alternatively, one can expand small sets of articles to display the most closely 
      related articles. We also implemented a novel cluster-by-topic method that
      generates a concise set of topics covering most of the retrieved articles.
      CONCLUSION: Anne O'Tate is an integrated, generic tool for summarization,
      drill-down and browsing of PubMed search results that accommodates a wide range
      of biomedical users and needs. It can be accessed at 4. Peer review and editorial
      matters for this article were handled by Aaron Cohen.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry and Psychiatric Institute, MC912, University of Illinois
      at Chicago, Chicago, IL 60612, USA. neils@uic.edu.
FAU - Zhou, Wei
AU  - Zhou W
FAU - Torvik, Vetle I
AU  - Torvik VI
LA  - eng
PT  - Journal Article
DEP - 20080215
PL  - England
TA  - J Biomed Discov Collab
JT  - Journal of biomedical discovery and collaboration
JID - 101258054
PMC - PMC2276193
OID - NLM: PMC2276193
EDAT- 2008/02/19 09:00
MHDA- 2008/02/19 09:01
CRDT- 2008/02/19 09:00
PHST- 2007/09/28 [received]
PHST- 2008/02/15 [accepted]
PHST- 2008/02/15 [aheadofprint]
AID - 1747-5333-3-2 [pii]
AID - 10.1186/1747-5333-3-2 [doi]
PST - epublish
SO  - J Biomed Discov Collab. 2008 Feb 15;3:2. doi: 10.1186/1747-5333-3-2.

PMID- 18812194
OWN - NLM
STAT- MEDLINE
DA  - 20081117
DCOM- 20090414
LR  - 20140921
IS  - 0168-0102 (Print)
IS  - 0168-0102 (Linking)
VI  - 62
IP  - 4
DP  - 2008 Dec
TI  - Natural antisense transcripts are co-expressed with sense mRNAs in
      synaptoneurosomes of adult mouse forebrain.
PG  - 236-9
LID - 10.1016/j.neures.2008.08.010 [doi]
AB  - Natural antisense transcripts and overlapping sense transcripts are expressed in 
      a variety of tissues, including adult mouse brain. Here we show that a subset of 
      mRNA-like sense-antisense transcript pairs are co-expressed within
      synaptoneurosomes of adult mouse forebrain, a subcellular fraction that is
      enriched in pinched-off dendritic spines of pyramidal neurons. Several of these
      pairs involve mRNAs that have been implicated in synaptic functions and in
      Alzheimer disease pathways. This study provides evidence that a new class of
      noncoding RNAs (natural antisense transcripts) are expressed near synapses, and
      encourages further studies of their roles in neuronal function.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry and Psychiatric Institute, University of
      Illinois-Chicago, 1601 W. Taylor Street MC912, Chicago, IL 60612, USA.
      neils@uic.edu
FAU - Lugli, Giovanni
AU  - Lugli G
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Mise, Nathan
AU  - Mise N
FAU - Ikeda, Rieko
AU  - Ikeda R
FAU - Abe, Kuniya
AU  - Abe K
LA  - eng
GR  - LM07292/LM/NLM NIH HHS/United States
GR  - MH81099/MH/NIMH NIH HHS/United States
GR  - R01 LM007292/LM/NLM NIH HHS/United States
GR  - R01 LM007292-05/LM/NLM NIH HHS/United States
GR  - R21 MH081099/MH/NIMH NIH HHS/United States
GR  - R21 MH081099-02/MH/NIMH NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20080904
PL  - Ireland
TA  - Neurosci Res
JT  - Neuroscience research
JID - 8500749
RN  - 0 (RNA, Antisense)
RN  - 0 (RNA, Messenger)
RN  - 0 (RNA, Untranslated)
RN  - 0 (Rab6 protein)
RN  - EC 3.4.- (Amyloid Precursor Protein Secretases)
RN  - EC 3.4.23.- (Aspartic Acid Endopeptidases)
RN  - EC 3.4.23.46 (BACE1 protein, human)
RN  - EC 3.5.1.- (Sirtuins)
RN  - EC 3.6.1.- (rab GTP-Binding Proteins)
SB  - IM
MH  - Amyloid Precursor Protein Secretases/genetics/metabolism
MH  - Animals
MH  - Aspartic Acid Endopeptidases/genetics/metabolism
MH  - Humans
MH  - Male
MH  - Mice
MH  - Mice, Inbred C57BL
MH  - Molecular Sequence Data
MH  - Prosencephalon/*ultrastructure
MH  - RNA, Antisense/genetics/*metabolism
MH  - RNA, Messenger/genetics/*metabolism
MH  - RNA, Untranslated/genetics/metabolism
MH  - Sirtuins/genetics/metabolism
MH  - Subcellular Fractions/metabolism
MH  - Synapses/genetics/*metabolism
MH  - Synaptosomes/*metabolism
MH  - rab GTP-Binding Proteins/genetics/metabolism
PMC - PMC2597790
MID - NIHMS80916
OID - NLM: NIHMS80916
OID - NLM: PMC2597790
EDAT- 2008/09/25 09:00
MHDA- 2009/04/15 09:00
CRDT- 2008/09/25 09:00
PHST- 2008/07/10 [received]
PHST- 2008/08/22 [revised]
PHST- 2008/08/26 [accepted]
PHST- 2008/09/04 [aheadofprint]
AID - S0168-0102(08)00239-3 [pii]
AID - 10.1016/j.neures.2008.08.010 [doi]
PST - ppublish
SO  - Neurosci Res. 2008 Dec;62(4):236-9. doi: 10.1016/j.neures.2008.08.010. Epub 2008 
      Sep 4.

PMID- 18410515
OWN - NLM
STAT- MEDLINE
DA  - 20080808
DCOM- 20080916
LR  - 20140903
IS  - 1471-4159 (Electronic)
IS  - 0022-3042 (Linking)
VI  - 106
IP  - 2
DP  - 2008 Jul
TI  - Expression of microRNAs and their precursors in synaptic fractions of adult mouse
      forebrain.
PG  - 650-61
LID - 10.1111/j.1471-4159.2008.05413.x [doi]
AB  - We have characterized the expression of microRNAs and selected microRNA
      precursors within several synaptic fractions of adult mouse forebrain, including 
      synaptoneurosomes, synaptosomes and isolated post-synaptic densities (PSDs),
      using methods of microRNA microarray, real time qRT-PCR, Northern blotting and
      immunopurification using anti-PSD95 antibody. The majority of brain microRNAs
      (especially microRNAs known to be expressed in pyramidal neurons) are detectably 
      expressed in synaptic fractions, and a subset of microRNAs is significantly
      enriched in synaptic fractions relative to total forebrain homogenate. MicroRNA
      precursors are also detectable in synaptic fractions at levels that are
      comparable to whole tissue. Whereas mature microRNAs are predominantly associated
      with soluble components of the synaptic fractions, microRNA precursors are
      predominantly associated with PSDs. For seven microRNAs examined, there was a
      significant correlation between the relative synaptic enrichment of the precursor
      and the relative synaptic enrichment of the corresponding mature microRNA. These 
      findings support the proposal that microRNAs are formed, at least in part, via
      processing of microRNA precursors locally within dendritic spines. Dicer is
      expressed in PSDs but is enzymatically inactive until conditions that activate
      calpain cause its liberation; thus, we propose that synaptic stimulation may lead
      to local processing of microRNA precursors in proximity to the synapse.
FAU - Lugli, Giovanni
AU  - Lugli G
AD  - Department of Psychiatry and Psychiatric Institute, University of Illinois at
      Chicago, Chicago, Illinois, USA.
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Larson, John
AU  - Larson J
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
LA  - eng
GR  - DC 05793/DC/NIDCD NIH HHS/United States
GR  - LM07292/LM/NLM NIH HHS/United States
GR  - MH81099/MH/NIMH NIH HHS/United States
GR  - R01 DC005793/DC/NIDCD NIH HHS/United States
GR  - R01 LM007292/LM/NLM NIH HHS/United States
GR  - R21 MH081099/MH/NIMH NIH HHS/United States
PT  - Historical Article
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20080412
PL  - England
TA  - J Neurochem
JT  - Journal of neurochemistry
JID - 2985190R
RN  - 0 (Intracellular Signaling Peptides and Proteins)
RN  - 0 (Membrane Proteins)
RN  - 0 (MicroRNAs)
RN  - 0 (RNA Precursors)
RN  - 0 (RNA, Messenger)
RN  - EC 2.7.4.8 (Dlgh4 protein, mouse)
RN  - EC 2.7.4.8 (Guanylate Kinase)
SB  - IM
MH  - Animals
MH  - Gene Expression Profiling/methods
MH  - Guanylate Kinase
MH  - History, Ancient
MH  - Immunoprecipitation/methods
MH  - Intracellular Signaling Peptides and Proteins/genetics/metabolism
MH  - Male
MH  - Membrane Proteins/genetics/metabolism
MH  - Mice
MH  - Mice, Inbred C57BL
MH  - MicroRNAs/genetics/*metabolism
MH  - Oligonucleotide Array Sequence Analysis/methods
MH  - Prosencephalon/*ultrastructure
MH  - RNA Precursors/genetics/*metabolism
MH  - RNA, Messenger/metabolism
MH  - Subcellular Fractions/metabolism
MH  - Synapses/*metabolism/*ultrastructure
MH  - Synaptosomes/*metabolism
PMC - PMC3711666
MID - NIHMS486824
OID - NLM: NIHMS486824
OID - NLM: PMC3711666
EDAT- 2008/04/16 09:00
MHDA- 2008/09/17 09:00
CRDT- 2008/04/16 09:00
PHST- 2008/04/12 [aheadofprint]
AID - JNC5413 [pii]
AID - 10.1111/j.1471-4159.2008.05413.x [doi]
PST - ppublish
SO  - J Neurochem. 2008 Jul;106(2):650-61. doi: 10.1111/j.1471-4159.2008.05413.x. Epub 
      2008 Apr 12.

PMID- 20072710
OWN - NLM
STAT- Publisher
DA  - 20100114
IS  - 1556-4681 (Print)
IS  - 1556-4681 (Linking)
VI  - 3
IP  - 3
DP  - 2009 Jul 1
TI  - Author Name Disambiguation in MEDLINE.
LID - 11 [pii]
AB  - BACKGROUND: We recently described "Author-ity," a model for estimating the
      probability that two articles in MEDLINE, sharing the same author name, were
      written by the same individual. Features include shared title words, journal
      name, coauthors, medical subject headings, language, affiliations, and author
      name features (middle initial, suffix, and prevalence in MEDLINE). Here we test
      the hypothesis that the Author-ity model will suffice to disambiguate author
      names for the vast majority of articles in MEDLINE. METHODS: Enhancements
      include: (a) incorporating first names and their variants, email addresses, and
      correlations between specific last names and affiliation words; (b) new methods
      of generating large unbiased training sets; (c) new methods for estimating the
      prior probability; (d) a weighted least squares algorithm for correcting
      transitivity violations; and (e) a maximum likelihood based agglomerative
      algorithm for computing clusters of articles that represent inferred
      author-individuals. RESULTS: Pairwise comparisons were computed for all author
      names on all 15.3 million articles in MEDLINE (2006 baseline), that share last
      name and first initial, to create Author-ity 2006, a database that has each name 
      on each article assigned to one of 6.7 million inferred author-individual
      clusters. Recall is estimated at ~98.8%. Lumping (putting two different
      individuals into the same cluster) affects ~0.5% of clusters, whereas splitting
      (assigning articles written by the same individual to >1 cluster) affects ~2% of 
      articles. IMPACT: The Author-ity model can be applied generally to other
      bibliographic databases. Author name disambiguation allows information retrieval 
      and data integration to become person-centered, not just document-centered,
      setting the stage for new data mining and social network tools that will
      facilitate the analysis of scholarly publishing and collaboration behavior.
      AVAILABILITY: The Author-ity 2006 database is available for nonprofit academic
      research, and can be freely queried via http://arrowsmith.psych.uic.edu.
FAU - Torvik, Vetle I
AU  - Torvik VI
AD  - University of Illinois at Chicago.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
LA  - ENG
GR  - R21 LM008364/LM/NLM NIH HHS/United States
GR  - R21 LM008364-02/LM/NLM NIH HHS/United States
PT  - JOURNAL ARTICLE
TA  - ACM Trans Knowl Discov Data
JT  - ACM transactions on knowledge discovery from data
JID - 101473241
PMC - PMC2805000
MID - NIHMS140129
EDAT- 2010/01/15 06:00
MHDA- 2010/01/15 06:00
CRDT- 2010/01/15 06:00
PST - ppublish
SO  - ACM Trans Knowl Discov Data. 2009 Jul 1;3(3). pii: 11.

PMID- 19185946
OWN - NLM
STAT- MEDLINE
DA  - 20090310
DCOM- 20090813
LR  - 20140901
IS  - 1872-7565 (Electronic)
IS  - 0169-2607 (Linking)
VI  - 94
IP  - 2
DP  - 2009 May
TI  - Arrowsmith two-node search interface: a tutorial on finding meaningful links
      between two disparate sets of articles in MEDLINE.
PG  - 190-7
LID - 10.1016/j.cmpb.2008.12.006 [doi]
AB  - The Arrowsmith two-node search is a strategy that is designed to assist
      biomedical investigators in formulating and assessing scientific hypotheses. More
      generally, it allows users to identify biologically meaningful links between any 
      two sets of articles A and C in PubMed, even when these share no articles or
      authors in common and represent disparate topics or disciplines. The key idea is 
      to relate the two sets of articles via title words and phrases (B-terms) that
      they share. We have created a free, public web-based version of the two-node
      search tool (http://arrowsmith.psych.uic.edu), have described its development and
      implementation, and have presented analyses of individual two-node searches. In
      this paper, we provide an updated tutorial intended for end-users, that covers
      the use of the tool for a variety of potential scientific use case scenarios. For
      example, one can assess a recent experimental, clinical or epidemiologic finding 
      that connects two disparate fields of inquiry--identifying likely mechanisms to
      explain the finding, and choosing promising follow-up lines of investigation.
      Alternatively, one can assess whether the existing scientific literature lends
      indirect support to a hypothesis posed by the user that has not yet been
      investigated. One can also employ two-node searches to search for novel
      hypotheses. Arrowsmith provides a service that cannot be carried out feasibly via
      standard PubMed searches or by other available text mining tools.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry and Psychiatric Institute, MC912, University of Illinois
      at Chicago, 1601W. Taylor Street, Chicago, IL 60612, USA. neils@uic.edu
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Zhou, Wei
AU  - Zhou W
LA  - eng
GR  - LM007292/LM/NLM NIH HHS/United States
GR  - LM008364/LM/NLM NIH HHS/United States
GR  - R01 LM007292/LM/NLM NIH HHS/United States
GR  - R01 LM007292-05/LM/NLM NIH HHS/United States
GR  - R21 LM008364/LM/NLM NIH HHS/United States
GR  - R21 LM008364-02/LM/NLM NIH HHS/United States
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
DEP - 20090130
PL  - Ireland
TA  - Comput Methods Programs Biomed
JT  - Computer methods and programs in biomedicine
JID - 8506513
SB  - IM
MH  - Abstracting and Indexing as Topic
MH  - Algorithms
MH  - *Information Storage and Retrieval
MH  - Internet
MH  - *MEDLINE
MH  - Natural Language Processing
MH  - Pattern Recognition, Automated
MH  - PubMed
MH  - Software
MH  - User-Computer Interface
PMC - PMC2693227
MID - NIHMS105821
OID - NLM: NIHMS105821
OID - NLM: PMC2693227
EDAT- 2009/02/03 09:00
MHDA- 2009/08/14 09:00
CRDT- 2009/02/03 09:00
PHST- 2008/09/07 [received]
PHST- 2008/10/17 [revised]
PHST- 2008/12/12 [accepted]
PHST- 2009/01/30 [aheadofprint]
AID - S0169-2607(08)00284-8 [pii]
AID - 10.1016/j.cmpb.2008.12.006 [doi]
PST - ppublish
SO  - Comput Methods Programs Biomed. 2009 May;94(2):190-7. doi:
      10.1016/j.cmpb.2008.12.006. Epub 2009 Jan 30.

PMID- 20309390
OWN - NLM
STAT- MEDLINE
DA  - 20100323
DCOM- 20110617
LR  - 20140827
IS  - 1759-0914 (Electronic)
VI  - 2
IP  - 1
DP  - 2010
TI  - Olfactory discrimination training up-regulates and reorganizes expression of
      microRNAs in adult mouse hippocampus.
PG  - e00028
LID - 10.1042/AN20090055 [doi]
AB  - Adult male mice (strain C57Bl/6J) were trained to execute nose-poke responses for
      water reinforcement; then they were randomly assigned to either of two groups:
      olfactory discrimination training (exposed to two odours with reward contingent
      upon correctly responding to one odour) or pseudo-training (exposed to two odours
      with reward not contingent upon response). These were run in yoked fashion and
      killed when the discrimination-trained mouse reached a learning criterion of 70% 
      correct responses in 20 trials, occurring after three sessions (a total of
      approximately 40 min of training). The hippocampus was dissected bilaterally from
      each mouse (N = 7 in each group) and profiling of 585 miRNAs (microRNAs) was
      carried out using multiplex RT-PCR (reverse transcription-PCR) plates. A
      significant global up-regulation of miRNA expression was observed in the
      discrimination training versus pseudo-training comparison; when tested
      individually, 29 miRNAs achieved significance at P = 0.05. miR-10a showed a
      2.7-fold increase with training, and is predicted to target several
      learning-related mRNAs including BDNF (brain-derived neurotrophic factor), CAMK2b
      (calcium/calmodulin-dependent protein kinase IIbeta), CREB1
      (cAMP-response-element-binding protein 1) and ELAVL2 [ELAV (embryonic lethal,
      abnormal vision, Drosophila)-like; Hu B]. Analysis of miRNA pairwise correlations
      revealed the existence of several miRNA co-expression modules that were specific 
      to the training group. These in vivo results indicate that significant, dynamic
      and co-ordinated changes in miRNA expression accompany early stages of learning.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Department of Psychiatry and UIC Psychiatric Institute MC912, University of
      Illinois at Chicago, 1601 W. Taylor Street, Chicago, IL 60612, USA. neils@uic.edu
FAU - Lugli, Giovanni
AU  - Lugli G
FAU - Lenon, Angela L
AU  - Lenon AL
FAU - Davis, John M
AU  - Davis JM
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Larson, John
AU  - Larson J
LA  - eng
PT  - Journal Article
DEP - 20100226
PL  - Unknown
TA  - ASN Neuro
JT  - ASN neuro
JID - 101507115
RN  - 0 (MicroRNAs)
SB  - IM
MH  - Animals
MH  - Behavior, Animal/physiology
MH  - Discrimination (Psychology)/*physiology
MH  - Hippocampus/*physiology
MH  - Learning/*physiology
MH  - Male
MH  - Mice
MH  - Mice, Inbred C57BL
MH  - MicroRNAs/genetics/*metabolism
MH  - Random Allocation
MH  - Smell/*physiology
MH  - Up-Regulation
PMC - PMC2832745
OID - NLM: PMC2832745
OTO - NOTNLM
OT  - BDNF, brain-derived neurotrophic factor
OT  - Ct, threshold cycle value
OT  - DHPG, (S)-3,5-dihydroxyphenylglycine
OT  - LTP, long-term potentiation
OT  - MEF2, myocyte enhancer factor-2
OT  - NMDA, N-methyl-d-aspartate
OT  - RISC, RNA-induced silencing complex
OT  - RT-PCR, reverse transcription-PCR
OT  - TLDA, TaqMan(R) Low Density Arrays
OT  - TOP, terminal oligopyrimidine
OT  - dicer
OT  - learning
OT  - miRNA, microRNA
OT  - microRNA
OT  - olfactory discrimination
OT  - pre-miR, miRNA small hairpin precursor
OT  - pri-miR, primary miRNA gene transcript
OT  - snoRNA, small nucleolar RNA
OT  - synaptic plasticity
EDAT- 2010/03/24 06:00
MHDA- 2011/06/18 06:00
CRDT- 2010/03/24 06:00
PHST- 2009/12/05 [received]
PHST- 2010/01/14 [revised]
PHST- 2010/01/18 [accepted]
PHST- 2010/02/26 [epublish]
AID - 10.1042/AN20090055 [doi]
PST - epublish
SO  - ASN Neuro. 2010 Feb 26;2(1):e00028. doi: 10.1042/AN20090055.

PMID- 21275079
OWN - NLM
STAT- MEDLINE
DA  - 20111020
DCOM- 20120730
IS  - 1469-5111 (Electronic)
IS  - 1461-1457 (Linking)
VI  - 14
IP  - 10
DP  - 2011 Nov
TI  - MicroRNA expression in rat brain exposed to repeated inescapable shock:
      differential alterations in learned helplessness vs. non-learned helplessness.
PG  - 1315-25
LID - 10.1017/S1461145710001628 [doi]
AB  - MicroRNA (miRNA) expression was measured within frontal cortex of male Holtzman
      rats subjected to repeated inescapable shocks at days 1 and 7, tested for learned
      helplessness (LH) at days 2 and 8, and sacrificed at day 15. We compared rats
      that did vs. did not exhibit LH, as well as rats that were placed in the
      apparatus and tested for avoidance but not given shocks (tested controls, TC).
      Non-learned helpless (NLH) rats showed a robust adaptive miRNA response to
      inescapable shock whereas LH rats showed a markedly blunted response. One set of 
      12 miRNAs showed particularly large, significant down-regulation in NLH rats
      relative to tested controls (mir-96, 141, 182, 183, 183*, 298, 200a, 200a*, 200b,
      200b*, 200c, 429). These were encoded at a few shared polycistronic loci,
      suggesting that the down-regulation was coordinately controlled at the level of
      transcription. Most of these miRNAs are enriched in synaptic fractions. Moreover,
      almost all of these share 5'-seed motifs with other members of the same set,
      suggesting that they will hit similar or overlapping sets of target mRNAs.
      Finally, half of this set is predicted to hit Creb1 as a target. We also
      identified a core miRNA co-expression module consisting of 36 miRNAs that are
      highly correlated with each other across individuals of the LH group (but not in 
      the NLH or TC groups). Thus, miRNAs participate in the alterations of gene
      expression networks that underlie the normal (NLH) as well as aberrant (LH)
      response to repeated shocks.
FAU - Smalheiser, Neil R
AU  - Smalheiser NR
AD  - Psychiatric Institute, Department of Psychiatry, University of Illinois at
      Chicago, Chicago, IL 60612, USA.
FAU - Lugli, Giovanni
AU  - Lugli G
FAU - Rizavi, Hooriyah S
AU  - Rizavi HS
FAU - Zhang, Hui
AU  - Zhang H
FAU - Torvik, Vetle I
AU  - Torvik VI
FAU - Pandey, Ghanshyam N
AU  - Pandey GN
FAU - Davis, John M
AU  - Davis JM
FAU - Dwivedi, Yogesh
AU  - Dwivedi Y
LA  - eng
GR  - R01MH 68777/MH/NIMH NIH HHS/United States
GR  - R01MH082802/MH/NIMH NIH HHS/United States
GR  - R21MH081099/MH/NIMH NIH HHS/United States
GR  - R21MH091509/MH/NIMH NIH HHS/United States
PT  - Comparative Study
PT  - Journal Article
PT  - Research Support, N.I.H., Extramural
PT  - Research Support, Non-U.S. Gov't
DEP - 20110128
PL  - England
TA  - Int J Neuropsychopharmacol
JT  - The international journal of neuropsychopharmacology / official scientific
      journal of the Collegium Internationale Neuropsychopharmacologicum (CINP)
JID - 9815893
RN  - 0 (MicroRNAs)
SB  - IM
MH  - Animals
MH  - *Behavior, Animal
MH  - Cerebral Cortex/*metabolism
MH  - Depression/etiology/*genetics/*psychology
MH  - Disease Models, Animal
MH  - Electroshock
MH  - Escape Reaction
MH  - Frontal Lobe/*metabolism
MH  - Gene Expression Profiling
MH  - Gene Expression Regulation
MH  - Gene Regulatory Networks
MH  - *Helplessness, Learned
MH  - Male
MH  - MicroRNAs/*metabolism
MH  - Rats
MH  - Rats, Sprague-Dawley
MH  - Reaction Time
MH  - Time Factors
MH  - Transcription, Genetic
EDAT- 2011/01/29 06:00
MHDA- 2012/07/31 06:00
CRDT- 2011/01/29 06:00
PHST- 2011/01/28 [aheadofprint]
AID - S1461145710001628 [pii]
AID - 10.1017/S1461145710001628 [doi]
PST - ppublish
SO  - Int J Neuropsychopharmacol. 2011 Nov;14(10):1315-25. doi:
      10.1017/S1461145710001628. Epub 2011 Jan 28.
